dotCMS/src/main/java/com/dotcms/contenttype/business/sql/FieldSql.java

```diff
@@ -23,22 +21,22 @@ public static FieldSql getInstance() {
 			+ "sort_order, field_values, regex_check, hint, default_value, field.fixed as fixed, field.read_only as read_only, "
 			+ "field.searchable as searchable, unique_, field.mod_date as mod_date, inode.inode as inode, owner, idate from inode, field ";
 
-	public String findById = SELECT_ALL_FIELDS + " where inode.inode = field.inode and inode.inode =?";
-	public String findByContentType = SELECT_ALL_FIELDS + " where inode.inode = field.inode and structure_inode =? order by sort_order";
-	public String findByContentTypeVar = SELECT_ALL_FIELDS
-			+ ", structure where inode.inode = field.inode and field.structure_inode = structure.inode and structure.velocity_var_name= ? order by sort_order";
-	public String findByContentTypeAndRelationType = SELECT_ALL_FIELDS
-			+ ", structure where inode.inode = field.inode and structure_inode =? and field.field_relation_type= ? order by sort_order";
+	public final String findById = SELECT_ALL_FIELDS + " where inode.inode = field.inode and inode.inode =?";
+	public final String findByContentType = SELECT_ALL_FIELDS + " where inode.inode = field.inode and structure_inode =? order by sort_order, velocity_var_name";
+	public final String findByContentTypeVar = SELECT_ALL_FIELDS
+			+ ", structure where inode.inode = field.inode and field.structure_inode = structure.inode and structure.velocity_var_name= ? order by sort_order, velocity_var_name";
+	public final String findByContentTypeAndRelationType = SELECT_ALL_FIELDS
+			+ ", structure where inode.inode = field.inode and structure_inode =? and field.field_relation_type= ? order by sort_order, velocity_var_name";
 	
 	
-	public String findByContentTypeAndFieldVar = SELECT_ALL_FIELDS + " where inode.inode = field.inode and structure_inode =? and field.velocity_var_name=?";
+	public final String findByContentTypeAndFieldVar = SELECT_ALL_FIELDS + " where inode.inode = field.inode and structure_inode =? and field.velocity_var_name=?";
 	
 	
-	public String deleteByContentType = "delete from field where structure_inode = ?";
-	public String deleteById = "delete from field where inode = ?";
-	public String deleteInodeById = "delete from inode where inode = ? and type='field'";
+	public final String deleteByContentType = "delete from field where structure_inode = ?";
+	public final String deleteById = "delete from field where inode = ?";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5572908)